lib/auth/v2/headerAuthCheck.js

```diff
@@ -15,7 +15,7 @@ function check(request, log, data) {
     timestamp = Date.parse(timestamp);
     if (!timestamp) {
         log.debug('missing security header: invalid date/timestamp');
-        return { err: errors.MissingSecurityHeader };
+        return { err: errors.AccessDenied };
```

Comment:
I do not know if this error can happen often, but when using ```customizeDescription()``` from ArsenalError we're creating a new ArsenalError and not using the same object like the "natives" ArsenalErrors. Can this lead to some loss of performances here ?